finetune/lora.py

```diff
@@ -108,9 +108,10 @@ def train(
         t0 = time.time()
 
         input_ids, targets = get_batch(fabric, train_data)
-        logits = model(input_ids)
-        loss = loss_fn(logits, targets)
-        fabric.backward(loss)
+        with fabric.no_backward_sync(model, enabled=((iter_num + 1) % gradient_accumulation_iters != 0)):
```

Comment:
FWY, I didn't add it here because this script was meant to be running on a single GPU. Anyway, doesn't hurt to have it.